File: languages/go/types/network_config.go
package types

type NetworkConfiguration struct {
	ContractAddress string `default:""`
	ContractABI     string `default:""`
	HttpProvider    string `default:""`
}

func NewNetworkConfiguration(contractAddress, contractABI, httpProvider string) NetworkConfiguration {
	return NetworkConfiguration{
		ContractAddress: contractAddress,
		ContractABI:     contractABI,
		HttpProvider:    httpProvider,
	}
}

File: languages/java/src/main/java/com/bloock/sdk/entity/NetworkConfig.java
package com.bloock.sdk.entity;

public class NetworkConfig {
  String contractAddress;
  String contractAbi;
  String httpProvider;

  public NetworkConfig(String contractAddress, String contractAbi, String httpProvider) {
    this.contractAddress = contractAddress;
    this.contractAbi = contractAbi;
    this.httpProvider = httpProvider;
  }

  public String getContractAddress() {
    return contractAddress;
  }

  public String getContractAbi() {
    return contractAbi;
  }

  public String getHttpProvider() {
    return httpProvider;
  }
}
